packages/web-twig/tests/Helper/TwigHelper.php

```diff
@@ -32,6 +34,7 @@ public static function setup(
 
         $loader->addPath(SpiritWebTwigExtension::DEFAULT_PARTIALS_PATH, SpiritWebTwigExtension::DEFAULT_PARTIALS_ALIAS);
         $loader->addPath(SpiritWebTwigExtension::DEFAULT_TWIG_COMPONENTS_PATH, SpiritWebTwigExtension::DEFAULT_PATH_ALIAS);
+        $loader->addPath(__DIR__ . '/../../static/', SpiritWebTwigExtension::DEFAULT_ICONS_ALIAS);
```

Comment:
It will be nice to have a comment here with something like `// Reference to a filesystem in Docker, please see packages/web-twig/docker/docker-compose.yml`